frontend/src/components/FacilityBooking/ViewBlock.tsx

```diff
@@ -19,15 +21,25 @@ const ViewBlock = (props: Props) => {
   }, [ref.current])
 
   return (
-    <StyledViewBooking ref={ref} isOccupied={props.entry.type === TimeBlockType.OCCUPIED} blockId={props.entry.id}>
-      {props.entry.type === TimeBlockType.OCCUPIED && (
-        <>
+    <>
+      {props.entry.booking ? (
```

Comment:
```suggestion
    props.entry.booking ? (
```
remove redundant empty tag